docs/api.swagger.json

```diff
@@ -2254,12 +2254,12 @@
           "x-go-name": "Public"
         },
         "redirect_uris": {
-          "description": "RedirectURIs is an array of allowed redirect urls for the client, for example: http://mydomain/oauth/callback .",
```

Comment:
This is weird, why is the description gone?

Reply:
I don't know, i assumed you'd removed it. Strange that's the only thing that changed. I tried running genswag again, same result. I'll investigate what I did wrong.

Reply:
Hm strange, maybe a version mismatch in go-swagger. The corresponding line is here: https://github.com/ory/hydra/blob/master/client/client.go#L39-L40

I think the issue is `, for example: http://mydomain/oauth/callback .` which is apparently parsed as an example section. You could try rewriting it to `, for example http://mydomain/oauth/callback`

Reply:
Damn, I was just a bit late! :D

Reply:
@arekkas fixed